apps/web/src/components/CurrencyInputPanel/index.tsx

```diff
@@ -216,6 +228,19 @@ const CurrencyInputPanel = memo(function CurrencyInputPanel({
                 />
               </Flex>
             ) : null}
+            {token &&
+            tokenAddress &&
+            token.chainId === ChainId.ZKSYNC &&
+            tokenAddress === '0x79db8c67d0c33203da4Efb58F7D325E1e0d4d692' ? (
```

Comment:
use `zksyncTokens.meow.address` will be better

Reply:
case sensitive compare?

Reply:
token address already checksumed. Should be good. Changed to use token instance to compare